services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusServiceITest.kt

```diff
@@ -893,6 +905,105 @@ class HakemusServiceITest(
+        fun `saves disclosure logs for the sent personal info`() {
+            val hanke = hankeFactory.saveWithAlue()
+            val founder = hankeKayttajaFactory.getFounderFromHanke(hanke)
+            val hakijaYhteystieto =
+                HakemusyhteystietoFactory.create(
+                    tyyppi = CustomerType.PERSON,
+                    nimi = "Ylevi Yhteyshenkilö",
+                    sahkoposti = "ylevi@hakemus.info",
+                    puhelinnumero = "111222333",
+                    ytunnus = null,
+                )
+            val asianhoitajaYhteystieto =
+                hakijaYhteystieto.copy(
+                    nimi = "Tytti Työläinen",
+                    sahkoposti = "tytti@hakeus.info",
+                    puhelinnumero = "999888777"
+                )
+            val hakemus =
+                hakemusFactory
+                    .builder(userId = "Other user")
+                    .withMandatoryFields()
+                    .hakija(hakijaYhteystieto, founder)
+                    .tyonSuorittaja(asianhoitajaYhteystieto, founder)
+                    .asianhoitaja()
+                    .save()
+            auditLogRepository.deleteAll()
+            every { cableReportService.create(any()) } returns alluId
+            every { cableReportService.getApplicationInformation(alluId) } returns
+                AlluFactory.createAlluApplicationResponse(alluId)
+
+            hakemusService.sendHakemus(hakemus.id, USERNAME)
+
+            val yhteystietoEntries = auditLogRepository.findAll()
+            assertThat(yhteystietoEntries).hasSize(5)
+            assertThat(auditLogRepository.findByType(ObjectType.ALLU_CUSTOMER)).hasSize(2)
+            assertThat(auditLogRepository.findByType(ObjectType.ALLU_CONTACT)).hasSize(3)
+            assertThat(yhteystietoEntries).each {
+                it.isSuccess(Operation.READ) {
+                    hasServiceActor("Allu")
+                    withTarget {
+                        hasId(hakemus.id)
+                        prop(AuditLogTarget::objectBefore).isNotNull()
+                        prop(AuditLogTarget::objectAfter).isNull()
+                    }
+                }
+            }
+            val expectedHakijaCustomer =
+                Customer(
+                    type = CustomerType.PERSON,
+                    name = "Ylevi Yhteyshenkilö",
+                    country = "FI",
+                    email = "ylevi@hakemus.info",
+                    phone = "111222333",
+                    registryKey = null,
+                    ovt = null,
+                    invoicingOperator = null,
+                    sapCustomerNumber = null
+                )
+            val expectedAsianhoitajaCustomer =
+                expectedHakijaCustomer.copy(
+                    name = "Tytti Työläinen",
+                    email = "tytti@hakeus.info",
+                    phone = "999888777",
+                )
+            val expectedPerustajaContact =
```

Comment:
Link to `founder` is not very clear. Neither are the used `DEFAULT_xxxxx` fields. One has to go check the factory code to see that the matching is correct.

Reply:
I changed this to use the founder's values. Is this better?